lib/routes.js

```diff
@@ -37,10 +37,7 @@ const responseSchema = {
             },
             extensions: {
               type: 'object',
-              properties: {
-                code: { type: 'string' },
-                timestamp: { type: 'string' }
-              }
```

Comment:
why were this removed?

Reply:
Hmm, yes I'll re-add `code`, but should `timestamp` stays? I think it shouldn't be hard-coded as a string. There might be an use-case where a developer want's `extensions.timestamp` to be a `number` type.